.ci/travis/travis.sh

```diff
@@ -116,7 +116,7 @@ jdk12)
   exclude3="\!TokenUtilTest#testTokenValueIncorrect2,"
   exclude4="\!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose"
 
-  mvn -e package -Passembly -Dtest=$exclude1$exclude2$exclude3$exclude4
+  mvn -e package -Passembly
```

Comment:
https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L1665
seems tests are skipped with the assembly profile.

Reply:
Strange that build is green , even it are failing https://travis-ci.org/checkstyle/checkstyle/jobs/597127920#L1353

Reply:
It seems to me we have seen something like this with JDK 12 before.

Reply:
We simply have mistakes in build commands, https://github.com/checkstyle/checkstyle/pull/7192 will fix them.